src/auth/googleauth.ts

```diff
@@ -111,6 +111,8 @@ export class GoogleAuth {
    * @private
    */
   private checkIsGCE?: boolean = undefined;
+  useJWTAccessAlways?: boolean;
+  defaultServicePath: string | undefined;
```

Comment:
If something is `type | undefined`, we typically use the `?` like we do elsewhere